src/featureflagservice/Dockerfile

```diff
@@ -20,7 +20,7 @@
 # Not until then.
 ARG ELIXIR_VERSION=1.14.3
 ARG OTP_VERSION=23.3.4.14
-ARG DEBIAN_VERSION=buster-20210902-slim
+ARG DEBIAN_VERSION=bullseye-20210902-slim
```

Comment:
issue: these have to be a valid tag of the [`hexpm/elixir`](https://hub.docker.com/r/hexpm/elixir) Docker image.

https://hub.docker.com/r/hexpm/elixir/tags

See `BUILDER_IMAGE`.

Reply:
They should be, I used the most recent debian build available for that elixir/otp version combination. The build passes, so the image was available to be pulled